frontend/src/components/Sidebar.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC } from "react";
+import { HomeIcon, FolderIcon, CogIcon, ShareIcon } from "lucide-react";
+import Image from "next/image";
+import { Button } from "./ui/button";
+
+const Sidebar: FC = () => {
+  return (
+    
+      <div className="relative flex flex-col  bg-white border-r group hover:w-52 w-[56px] w-min-[56px] transition-all  duration-300 ease-in-out overflow-hidden" style={{
+        height: "calc(100vh - 64px)"
+      }}>
+        
+        <div className="flex flex-col flex-1 p-2 space-y-4">
```

Comment:
Maybe let's use gap-# for flexboxes to be consistent and stuff
```suggestion
        <div className="flex flex-col flex-1 p-2 gap-4">
```

Reply:
i think i will leave it, probably i had some bugs related to it

Reply:
Sure